common/locales/de-CH-scout.json

```diff
@@ -1,2 +1 @@
-{
-}
+{}
```

Comment:
The format of the translation jsons is already managed by an automated tool, namely lokalise. Can you disable prettier for these files? Otherwise we will have back-and-forth formatting changes after every translation update.

Reply:
Fixed in b1acdb3